.github/actions/dbp-charts/verify-compose/docker_compose.sh

```diff
@@ -5,6 +5,26 @@ COMPOSE_PATH=$(dirname $COMPOSE_FILE_PATH)
 COMPOSE_BIN="docker compose"
 alf_port=8080
 
+containers_dump_logs_on_error() {
```

Comment:
I would use different name - is it only dumping logs when on error?

Reply:
indeed not, it's only being called on errors in the script XD